dockerfile_functions.sh

```diff
@@ -76,32 +76,25 @@ EOI
 # Select the alpine OS packages.
 # Install GNU glibc as this OpenJDK build is compiled against glibc and not musl.
 print_alpine_pkg() {
-	cat >> $1 <<'EOI'
+	cat >> "$1" <<EOI
+COPY --from=$target_registry/openjdk-glibc-builder:2.28-$(uname -m) /usr/glibc-compat/. /usr/glibc-compat/
+COPY --from=$target_registry/openjdk-gcc-builder:8.2.0-$(uname -m) /usr/glibc-compat/. /usr/glibc-compat/lib/
+COPY --from=$target_registry/openjdk-zlib-builder:1.2.11-$(uname -m) /usr/glibc-compat/. /usr/glibc-compat/lib/
+
```

Comment:
Also -- all those dependency images must be available in repository before we can start building using dockerfiles. Any plan to publish them ?